### ISSUE TITLE
关于solver.py中prior_loss、series_loss的计算

### ISSUE THREAD

**Issue by Alex-seven** [thuml/Anomaly-Transformer#9]
prior_loss和series_loss计算用到的变量相同，本来就是这样的还是手误呐？

**Comment by wuhaixu2016** [comment]
感谢关注！
目前代码没有问题，如果你感觉哪里不对劲的话，可以具体到行吗？

**Comment by Alex-seven** [comment]
> 
在方法train中:
series_loss +=   troch.mean(kl_loss(S, P))   +  torch.mean(kl_loss(P, S))
prior_loss +=   torch.mean(kl_loss(P, S))   +   torch.mean(kl_loss(S, P)) 
这样的话，series_loss与prior_loss就是A+B和B+A的区别咯，那就相等了呀

**Comment by catmonkeylee** [comment]
目前我看代码以及程序验证，确实 series_loss == prior_loss ；期待作者 @wuhaixu2016 能帮忙解答下

for u in range(len(prior)):
                series_loss += (
                    torch.mean(my_kl_loss(series[u],
                                          (prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1, self.win_size)).detach())) +
                    torch.mean(my_kl_loss(
                        (prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1, self.win_size)).detach(), series[u])))
                prior_loss += (
                    torch.mean(my_kl_loss(
                        (prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1, self.win_size)), series[u].detach())) +
                    torch.mean(my_kl_loss(series[u].detach(),
                                          (prior[u] / torch.unsqueeze(torch.sum(prior[u], dim=-1), dim=-1).repeat(1, 1, 1, self.win_size)))))
